app/routes/$bandId.setlists.sortBy.tsx

```diff
@@ -16,8 +16,21 @@ import {
   Label,
   RadioGroup,
 } from "~/components";
+import { userPrefs } from "~/models/cookies.server";
 import { requireUserId } from "~/session.server";
 
+export async function loader({request, params}: LoaderFunctionArgs) {
+  await requireUserId(request);
+  const { bandId } = params;
+  invariant(bandId, "bandId not found");
+
+  const cookieHeader = request.headers.get("Cookie");
```

Comment:
these three lines could probably be pulled into a helper func if more userPrefs were created in the future.